wrapping/itkTorchImage.wrap

```diff
@@ -1,4 +1,4 @@
-set(WRAP_ITK_TORCH_SCALAR "F" "D" "UC" "SC" "SS" "SL" "SLL")
+set(WRAP_ITK_TORCH_SCALAR "F" "D" "B" "UC" "SC" "SS" "SL" "SLL")
```

Comment:
Is `B` supported and interchangeable with `UC`?

Reply:
[This documentation page](https://pytorch.org/docs/stable/tensors.html) indicates that `torch.bool` is distinct from `torch.uint8` and from `torch.int8`, at least at the interface level.  I don't know how Torch handles the underlying implementation.

Reply:
I thought that B is for `byte`, not `bool` 😄